src/components/dialogs/VerifyEmailDialog.tsx

```diff
@@ -235,7 +260,32 @@ export function Inner({
           </View>
         ) : null}
         <View style={[a.gap_sm, gtMobile && [a.flex_row_reverse, a.ml_auto]]}>
-          {currentStep === 'StepOne' ? (
+          {currentStep === 'Reminder' ? (
+            <>
+              <Button
+                label={_(msg`Get started`)}
+                variant="solid"
+                color="primary"
+                size="large"
+                onPress={() => setCurrentStep('StepOne')}>
+                <ButtonText>
+                  <Trans>Get started</Trans>
+                </ButtonText>
+              </Button>
+              <Button
+                label={_(msg`Maybe layer`)}
+                accessibilityHint={_(msg`Snooze reminder`)}
```

Comment:
```suggestion
                accessibilityHint={_(msg`Snoozes reminder`)}
```
I think `Snoozes` works better because IIRC `accessibilityHint` describes the result of the action? Not completely sure on that though tbh